### ISSUE TITLE
fedora-logos conflict with fedberry-logos

### ISSUE THREAD

**Issue by fvzwieten** [fedberry/fedberry#34]
This is on F25:

dnf install php
.
.
Error: Transaction check error:
  file /usr/share/pixmaps/poweredby.png from install of fedora-logos-httpd-22.0.0-3.fc24.noarch conflicts with file from package fedberry-logos-24.0-1.fc25.noarch

**Comment by agrez** [comment]
Thanks for this report. I'll drop /usr/share/pixmaps/poweredby.png which should fix the issue.

**Comment by agrez** [comment]
BTW, I haven't done any f25 releases yet (or even tried updating from 24---->25). I am suprised anything works! :-)

**Comment by fvzwieten** [comment]
Well, somebody did try before me and it showed up in my feedreader from planet Fedora, so I just gave it a try, because I want to use php7 / mariadb / lighttpd combo for nextcloud box.

See https://www.montanalinux.org/fedberry-25-on-raspberry-pi-3.html

**Comment by agrez** [comment]
Glad it worked out for him, it seems his bravery has paid off. I have installed and have been using Fedberry 25 for a while now (fresh install from unreleased images I have been building internally) but haven't tried updating from 24---->25. Once I get some time to iron out the remaining selinux issues, I will push up some images.

**Comment by fvzwieten** [comment]
It worked out for me as well. Speaking about selinux though: I installed and enabled it after the move to F25 (I am on barebone which did not have selinux by default), it freezed during boot. After someone from #selinux irc channel helped me out it worked again. If you want, I can share with you what we did to make it work.

I played with dietpi before fedberry and was amazed fedberry barebone image's footprint (30MB) was well below dietpi (37 MB) when compared using htop. However, that was before I enabled selinux which added a whopping 20 MB to this. That are some massive policies loaded up there!
